index.d.ts

```diff
@@ -0,0 +1,67 @@
+    // Type definitions for parsel-js 1.0.1
+    // Project: https://github.com/LeaVerou/parsel
+    // Definitions by: Naveen DA <https://github.com/NaveenDA>
+    // Definitions: https://github.com/LeaVerou/parsel/blob/master/index.d.ts
```

Comment:
The entire file appears to be indented? Also, would it be possible to use tabs for indentation, like the rest of the project?

Reply:
Yeah, it seems like indented. Now fixed